src/main/java/EventScheduler.java

```diff
@@ -0,0 +1,94 @@
+package main.java;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EventScheduler {
+    private ArrayList<Event> eventlst;
+    private TaskToEvent converter;
+    private GapFinder gapFinder;
+
+    public EventScheduler(TaskToEvent obj, GapFinder gapFinder){
+        this.eventlst = new ArrayList<Event>();
+        this.converter = obj;
+        this.gapFinder = gapFinder;
+    }
+
+    public void addEvents(Event event){
+        this.eventlst.add(event);
+    }
+
+    public void removeEvents(Event event){
+        this.eventlst.remove(event);
+    }
+
+    /**  IS THIS NECESSARY????????????????
+    public void addTaskToTodoList(){
+        for(Event evt: this.eventlst){
+            this.todolst.addTask(evt.getTask());
```

Comment:
I removed the  todolist that was in CalendarBuilder but I can add it back in if we need it. I just wasn't sure what it did